script.js

```diff
@@ -1,4 +1,460 @@
+var hitOrStand = "Ask player to hit or stand.";
+var startDealing = "shuffle cards and start dealing";
+var nameRequest = "request for player game";
```

Comment:
Try to name your modes in upper camel case like `NAME_REQUEST` to for clarity and it is according to the convention.